akka-cluster-sharding/src/test/scala/akka/cluster/sharding/GetShardTypeNamesSpec.scala

```diff
@@ -37,7 +37,7 @@ class GetShardTypeNamesSpec extends AkkaSpec(GetShardTypeNamesSpec.config) with
 
   "GetShardTypeNames" must {
     "contain empty when join cluster without shards" in {
-      ClusterSharding(system).shardTypeNames should ===(Set())
+      ClusterSharding(system).shardTypeNames should ===(Set.empty[String])
```

Comment:
Hehe, nice with the previous "compare this but only if it is the exact same type" and then "oh, by the way, infer this right hand type for me before you compare" :D